pkg/controllers/node/hostendpoints.go

```diff
@@ -28,6 +28,44 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	defaultProfileName = "allow"
+)
+
+// createAllowProfile creates an allow-all profile  that is used by
+// the hostendpoints created by this controller. This is only done for etcd
+// backed clusters (for kubernetes datastore, we just return the kvp defined in
+// libcalico-go).
+func (c *NodeController) createAllowProfile() error {
+	if c.config.DatastoreType != "kubernetes" {
```

Comment:
this is going to conflict with my PR, as I don't pass the DatastoreType to NodeController anymore